static/src/js/schemas/uiSchemas/services/operationMetadata.js

```diff
@@ -0,0 +1,843 @@
+                        'ui:group-classname': 'h3-title',
+                        'ui:col': {
+                          children: [
+                            {
+                              'ui:row': [
+                                {
+                                  'ui:col': {
+                                    md: 6,
+                                    children: ['Latitude']
+                                  }
+                                },
+                                {
+                                  'ui:col': {
+                                    md: 6,
+                                    children: ['Longitude']
+                                  }
+                                }
+                              ]
+                            }
+                          ]
+                        }
+                      }
+                    ]
+                  },
+                  Latitude: {
+                    'ui:type': 'number'
+                  },
+                  Longitude: {
+                    'ui:type': 'number'
+                  }
+                },
+                EndPoint: {
+                  'ui:field': 'layout',
+                  'ui:layout_grid': {
+                    'ui:row': [
+                      {
+                        'ui:col': {
+                          children: [
+                            {
+                              'ui:group': 'End Point',
+                              'ui:group-description': true,
+                              'ui:group-classname': 'h3-title',
+                              'ui:row': [
+                                {
+                                  'ui:col': {
+                                    md: 6,
+                                    children: ['Latitude']
+                                  }
+                                },
+                                {
+                                  'ui:col': {
+                                    md: 6,
+                                    children: ['Longitude']
+                                  }
+                                }
+                              ]
+                            }
+                          ]
+                        }
+                      }
+                    ]
+                  },
+                  Latitude: {
+                    'ui:type': 'number'
+                  },
+                  Longitude: {
+                    'ui:type': 'number'
+                  }
+                }
+              }
+            },
+            SpatialBoundingBox: {
+              'ui:field': 'layout',
+              'ui:layout_grid': {
+                'ui:row': [
+                  {
+                    'ui:col': {
+                      children: [
+                        {
+                          'ui:group': 'Bounding Box',
+                          'ui:group-description': true,
+                          'ui:group-classname': 'h3-title',
```

Comment:
remove, should of said this way back but group-classname and group-box-classname are done